docs/hello-neighbour.yaml

```diff
@@ -15,6 +15,15 @@ tags:
   description: "Everything relating to church members interacting with the system"
 schemes:
 - "https"
+securityDefinitions:
+  jwtAuth:
+    type: apiKey
+    name: X-JWT
+    in: header
```

Comment:
I don't think this exists in Swagger 2.0. OpenAPI 3.0 supports it but the code generators struggle with that version.

Reply:
Marking this as resolved, as I see you have adjusted the security definition slightly. Fortunately Swagger 2.0 supports some types of Authorization where we can mimic Bearer